packages/react/src/components/Popover/index.js

```diff
@@ -111,9 +247,14 @@ const PopoverContent = React.forwardRef(function PopoverContent(
   ref
 ) {
   const prefix = usePrefix();
+  const { floating } = React.useContext(PopoverContext);
+
+  const mergedRef = useMergedRefs([floating, ref]);
```

Comment:
Super small nitpick, the convention I think I've been using is:

```suggestion
  forwardRef
) {
  const prefix = usePrefix();
  const { floating } = React.useContext(PopoverContext);
  const ref = useMergedRefs([floating, forwardRef]);
```